vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/profile.properties

```diff
@@ -1,7 +1,7 @@
 bdd.all-meta-filters=groovy: (!layout || layout.contains('desktop')) && !skip && (${bdd.meta-filters})
 
-selenium.grid.platform-name=Windows
-selenium.grid.platform-version=10
+selenium.grid.platform-name=
+selenium.grid.platform-version=
```

Comment:
which platform is used in case of empty values by:
- SauceLabs
- BrowserStack
- LambdaTest
- custom Selenium Grid
?

Reply:
I believe this is a dynamic value and will change over time.  For now it seems:
LT: 10
BS: 8.1 (quite strange)
SL: 7 (veeeery strange)

I don't get the question about the custom grid